src/gtp/mod.rs

```diff
@@ -143,14 +166,20 @@ impl<'a> GTPInterpreter<'a> {
             },
             "genmove"          => {
                 let color = Color::from_gtp(command[1]);
-                let m  = self.engine.gen_move(color, &self.game);
+                self.timer.start();
+                let budget = self.timer.budget(&self.game);
+                let mut stream = stderr();
+                stream.write_line(format!("Thinking for {}ms", budget).as_slice());
+                stream.write_line(format!("{}ms time left", self.timer.main_time_left()).as_slice());
```

Comment:
Yes, I'd prefer to leave it in for now. GTP ignores stderr so everything works just fine. And when I run a game against GnuGo in GoGui I can directly see how long I will have to wait for the next move. And then I can see if it makes sense. After all it's possible to change the constant (i.e. use something other than `0.5`) or even use a completely different formula.